apisix/plugins/request-id.lua

```diff
@@ -245,5 +245,8 @@ function _M.init()
     end
 end
 
+function _M.destroy()
+    timers.unregister_timer("plugin#request-id")
```

Comment:
Better to check if the timer is registered?

Reply:
Is it necessary to do this?  I have not seen similar examples elsewhere.

Reply:
If we don't register a timer, why unregister it?

Reply:
You have a point, I added the logic of judgment.